src/Tests/dotnet-workload-install.Tests/GivenWorkloadManifestUpdater.cs

```diff
@@ -674,7 +674,48 @@ public void TestSideBySideUpdateChecks()
             File.GetLastAccessTime(sentinelPath2).Should().BeCloseTo(updateTime2);
         }
 
-       
+        /**
```

Comment:
Is this test needed? If you want to keep the code and just ignore the test you can use `[Theory(Skip="some reason ")]`